hotdoc/core/tree.py

```diff
@@ -100,9 +100,11 @@ class Page:
                    Optional('render-subpages'): bool,
                    Optional('auto-sort'): bool,
                    Optional('full-width'): bool,
-                   Optional('see-also'): And(str, len),
+                   Optional('see-also'): str,
```

Comment:
why?

Reply:
You can have empty strings, often happens in Gst.

Reply:
Right, I see that in eg. -bad/ext/curl/gstcurlbasesink.c, but IMHO that's a documentation bug :)